src/backend/distributed/commands/function.c

```diff
@@ -626,6 +644,18 @@ UpdateFunctionDistributionInfo(const ObjectAddress *distAddress,
 		isnull[Anum_pg_dist_object_colocationid - 1] = true;
 	}
 
+	replace[Anum_pg_dist_object_force_pushdown - 1] = true;
+	if (forcePushdownAddress != NULL)
```

Comment:
I think we could always set forcePushdownAddress in this function? Given that we know we're dealing with a function.

Reply:
Maybe I didn't understand your comment correctly. The 'forcePushdownAddress' is a parameter passed by the 3 different callers. It could be a NULL (where user didn't pass the parameter in create_distribute_function..) or an address holding an explicit true or false. If we set it locally, what value we should set it to?

Reply:
If user specifies true/false explicitly, the values will be true/false. If the user skips the argument, it will be NULL.
In the UDF we can change this
                                                       force_pushdown bool DEFAULT NULL)
to
                                                       force_pushdown bool DEFAULT false)